### PR TITLE
Remove Internet Explorer 11 specific logic from KTextTruncator 

### PR DESCRIPTION
**Pull request by Nivas7**
Remove Internet Explorer 11 specific logic from KTextTruncator - #643

## Description
In summary, to remove IE11 support from the KTextTruncator component:

Remove the lineHeightIE prop.
Remove the IE11 fallback logic and related styles from the truncate method.

This change, treated as a breaking change, aims to simplify the component by removing unnecessary code for the unsupported IE11 browser. The updated component should be thoroughly tested to ensure no regressions.

Addresses #*PR# HERE*

### Before/after screenshots

Before
image

After
image

## Changelog

- [#643]
  - **Description:** Summary of change(s)
  - **Products impact:** Choose from - none (for internal updates) / bugfix / new API / updated API / removed API. If it's 'none', use "-" for all items below to indicate they are not relevant.
  - **Addresses:** Link(s) to GH issue(s) addressed. Include KDS links as well as links to related issues in a consumer product repository too.
  - **Components:** Affected public KDS component. Do not include internal sub-components or documentation components.
  - **Breaking:** Will this change break something in a consumer? Choose from: yes / no
  - **Impacts a11y:** Does this change improve a11y or adds new features that can be used to improve it? Choose from: yes / no
  - **Guidance:** Why and how to introduce this update to a consumer? Required for breaking changes, appreciated for changes with a11y impact, and welcomed for non-breaking changes when relevant.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included

## Reviewer guidance

- [x] Is the code clean and well-commented?

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

**Comment by Nivas7** [comment]
I hope this Problem Will be solved?

File: lib/KTextTruncator.vue [hunk 1]
```diff
@@ -1,19 +1,4 @@
 <template>
-
-  <!--
```
Comment (MisRob) [inline lib/KTextTruncator.vue:3]: It's fine to cleanup comments that are specific to IE11, but please make sure to read them carefully and do not remove comments that have nothing to do with IE11 logic. This applies to all other removed comments in this file.

**Review by MisRob — CHANGES_REQUESTED** [review]
Hey @Nivas7, thanks! Please see my note about comments. Resolving it should also help with missing "Props" descriptions for `text` and `maxLines` (can be seen in the "After" screenshots)

**Comment by MisRob** [comment]
Also please run linter

**Comment by Nivas7** [comment]
I will make that changes quickly:)

File: lib/KTextTruncator.vue (hunk 1 above)
Comment (Nivas7) [reply lib/KTextTruncator.vue:3]: That's Good Catch!

**Comment by Nivas7** [comment]
apologize for the mistake i re fork my repo and made my pr

**Comment by Nivas7** [comment]
How can i made pull req to develop barnch from fork it has only releases channellog.md  not develop branch challange.md

**Comment by Nivas7** [comment]
I solved the linting error and removed the Internet explorer props and fallback function i got so much confusion in pushing this to develop Branch , I Apologized for this

**Comment by MisRob** [comment]
Hi @Nivas7, no problem. I'd recommend you pull the latest develop from upstream to your local, start a new branch from it, and then open a new pull request. It may be helpful to spend a while looking into some tutorials on git. If you don't see the `upstream`'s develop on your forked repository, this could help https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository

**Pull request closed without merging**